README.md

```diff
@@ -101,15 +101,46 @@ dot -Tpdf ParserImpl.dot > ParserImpl.pdf
 
 # Getting started
 
-## Installing p4c from a Debian repository
-p4c has package support for several Ubuntu distributions (Ubuntu 20.04 to Ubuntu 21.04).
-It can be installed by adding the following ppa:
+## Installing packaged versions of p4c
+
+p4c has package support for several Ubuntu and Debian distributions.
+
+### Ubuntu
+
+For Ubuntu 20.04 and Ubuntu 21.04 it can be installed as follows:
+
+```bash
+. /etc/os-release
```

Comment:
Is this a typo?

Reply:
No, it is not a typo.
`.` is equivalent to `source` in shell